### PR TITLE
fix(sdk): Disable the `receipts` extension temporarily

### PR DESCRIPTION
**Pull request by Hywan**
Address #1911.

Please see https://github.com/matrix-org/matrix-rust-sdk/issues/2037.

### CODE REVIEW

**Review by stefanceriu — APPROVED** [review]
🚢 it!

**Review by bnjbvr — APPROVED** [review]
Oh wow, thought I had done it recently. Thanks!

File: crates/matrix-sdk/src/sliding_sync/builder.rs [hunk 1]
```diff
@@ -128,9 +128,13 @@ impl SlidingSyncBuilder {
                 cfg.account_data.enabled = Some(true);
             }
 
+            // TODO: Re-enable once a `lists` and `rooms` will support `*`. Seeo
```
Comment (poljar) [inline crates/matrix-sdk/src/sliding_sync/builder.rs:131]:
```suggestion
            // TODO: Re-enable once a `lists` and `rooms` will support `*`. See
```

File: crates/matrix-sdk/src/sliding_sync/builder.rs [hunk 2]
```diff
@@ -128,9 +128,13 @@ impl SlidingSyncBuilder {
...
+            // https://github.com/matrix-org/matrix-rust-sdk/issues/2037 for the tracking.
```
Comment (poljar) [inline crates/matrix-sdk/src/sliding_sync/builder.rs:132]:
```suggestion
            // https://github.com/matrix-org/matrix-rust-sdk/issues/2037 for more info.
```

**Review by poljar — APPROVED** [review]

File: crates/matrix-sdk/src/sliding_sync/builder.rs (hunk 2 above)
Comment (Hywan) [reply crates/matrix-sdk/src/sliding_sync/builder.rs]: :+1:

File: crates/matrix-sdk/src/sliding_sync/builder.rs (hunk 1 above)
Comment (Hywan) [reply crates/matrix-sdk/src/sliding_sync/builder.rs]: :+1:

**Pull request merged**